lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -782,7 +782,7 @@ void OCLBackend::doForwardPass() {
 
       setKernelArg<cl_uint>(kernel, 5, CC->getKernel());
       setKernelArg<cl_uint>(kernel, 6, CC->getStride());
-      setKernelArg<cl_uint>(kernel, 7, CC->getPad());
+      setKernelArg<cl_uint>(kernel, 7, CC->getPads()[0]);
```

Comment:
Add a comment saying that all the padding must be the same for OpenCL so the one we take does matter.
Basically, document why [0] is what we want.

Reply:
btw, why is this the case :)? (same paddings for all directions).

Reply:
@rdzhabarov So far, OpenCL doesn't support non-square padding yet. Will leave non-square padding for OpenCL to next PR since I need to understand OpenCL conv algorithm first :)

Reply:
BTW, I think this is a dead code path.
Normally, ConvolutionNode is converted to OCLConvolutionNode, and then to OCLConvolutionInst. So OpenCL should never see ConvolutionInst.
We may want to investigate in future PRs.